rest/example/launchpad/main.go

```diff
@@ -0,0 +1,35 @@
+package main
+
+import (
```

Comment:
It might be worth adding another command to the Makefile for this. We have `make rest-example` and `make ws-example` so maybe something like `make launchpad-example`. Up to you if you want to add it though

Reply:
added